pkg/server/server.go

```diff
@@ -70,6 +69,8 @@ func NewServer(ctx context.Context, sctx *sctx.Context) (srv *Server, err error)
 	}
 	cfg := srv.ConfigManager.GetConfig()
 
+	printInfo(lg)
```

Comment:
Add some comments. I'm afraid someone will revert it in the future.

Reply:
Done